FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -131,44 +188,16 @@ public async Task<IEnumerable<Job>> GetActiveJobsAsync(CancellationToken cancell
             return jobs;
         }
 
-        public async Task UpdateJobAsync(Job job, CancellationToken cancellationToken = default)
-        {
-            EnsureArg.IsNotNull(job, nameof(job));
-
-            job.LastHeartBeat = DateTimeOffset.UtcNow;
-
-            try
-            {
-                await _blobContainerClient.UpdateBlobAsync(
-                    $"{AzureBlobJobConstants.ActiveJobFolder}/{job.Id}.json",
-                    job.ToStream(),
-                    cancellationToken: cancellationToken);
-
-                _logger.LogInformation("Update job '{job.Id}' successfully.", job.Id);
-            }
-            catch (Exception ex)
-            {
-                _logger.LogError(ex, "Failed to update job '{job.Id}'", job.Id);
-                throw;
-            }
-        }
-
-        public async Task CompleteJobAsync(Job job, CancellationToken cancellationToken = default)
+        private async Task CompleteJobInternal(Job job, bool releaseLock, CancellationToken cancellationToken)
```

Comment:
Nit: `CompleteJobAsyncInternal`? (And maybe adding `Async` for other functions as well?)

Reply:
Thanks! Updated